helicopter.py

```
@@ -0,0 +1,37 @@
+import requests
+import pandas as pd
+from bs4 import BeautifulSoup
+
+# Wikipedia page URL
+url = "https://en.wikipedia.org/wiki/List_of_helicopter_prison_escapes"
+
+# Use headers to avoid being blocked
+headers = {"User-Agent": "Mozilla/5.0"}
+
+response = requests.get(url, headers=headers)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)